sdk/analytics/src/main/java/com/klaviyo/analytics/model/ProfileKey.kt

```diff
@@ -58,3 +58,10 @@ sealed class ProfileKey(name: String) : Keyword(name) {
         else -> name
     }
 }
+
+internal val PROFILE_IDENTIFIERS = arrayOf(
```

Comment:
I wanted to put this inside `Profile.companionObject` but there must be some initialization order issue, because it just came out as an array of nulls.